kamailio/registrar-role.cfg

```diff
@@ -281,11 +281,12 @@ route[SAVE_LOCATION]
     }
 
     $var(ip) = $Ri;
+    $var(port) = $Rp;
     if(af==INET6) {
        $var(ip) = "[" + $Ri + "]";
     }
 
-    $var(amqp_payload_request) = '{"Event-Category" : "directory", "Event-Name" : "reg_success", "Status" : "$var(Status)", "Event-Timestamp" : $TS, "Expires" : $(var(expires){s.int}), "First-Registration" : $var(new_reg), "Contact" : "$(ct{s.escape.common})", "Call-ID" : "$ci", "Realm" : "$fd", "Username" : "$fU", "From-User" : "$fU", "From-Host" : "$fd", "To-User" : "$tU", "To-Host" : "$td", "User-Agent" : "$(ua{s.escape.common})" , "Custom-Channel-Vars" : $xavp(ulattrs=>custom_channel_vars), "Proxy-Path" : "sip:$var(ip)", "RUID" : "$xavp(ulrcd=>ruid)" }';
+    $var(amqp_payload_request) = '{"Event-Category" : "directory", "Event-Name" : "reg_success", "Status" : "$var(Status)", "Event-Timestamp" : $TS, "Expires" : $(var(expires){s.int}), "First-Registration" : $var(new_reg), "Contact" : "$(ct{s.escape.common})", "Call-ID" : "$ci", "Realm" : "$fd", "Username" : "$fU", "From-User" : "$fU", "From-Host" : "$fd", "To-User" : "$tU", "To-Host" : "$td", "User-Agent" : "$(ua{s.escape.common})" , "Custom-Channel-Vars" : $xavp(ulattrs=>custom_channel_vars), "Proxy-Path" : "sip:$var(ip):$var(port)", "RUID" : "$xavp(ulrcd=>ruid)" }';
```

Comment:
@lazedo Thanks for the lightning-fast reply! Sometimes I wonder if you ever sleep :)

Kamailio is indeed performing it's call routing function properly. What Kamailio is not doing is publishing the correct registration contact port via AMQP in our use case: 

1. `SRV` domains are used.
2. outbound proxies are not used.

I must apologies for my lack of knowledge on Kamailio configuration. Does this PR address this problem by getting Kamailio to publish the registration contact port to AMQP?

If this PR doesn't fix this problem, or there is a more elegant way to solve the problem we're having, maybe we could open a Jira ticket? 

How would you like to proceed?